pkg/controller/generic_reconciler.go

```diff
@@ -63,6 +73,11 @@ func (gr *GenericReconciler) Reconcile(request reconcile.Request) (reconcile.Res
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	reqLogger.V(2).Info("Reconcile", "Kind", gr.reconciledKind)
 
+	if namespaceIgnore != nil && namespaceIgnore.Match([]byte(request.Namespace)) {
+		reqLogger.Info("Ignoring object as it matches namespace ignore pattern")
```

Comment:
It would be helpful to print the namespace/name/kind of the object.  It's always possible the regex is wrong and is excluding objects that are actually desired to be watched.  Proving logging as to what is excluded will help debug.

Reply:
This is how it gets printed currently: `{"level":"info","ts":1601385823.4955745,"logger":"DeploymentController","msg":"Ignoring object as it matches namespace ignore pattern","Request.Namespace":"openshift-apiserver-operator","Request.Name":"openshift-apiserver-operator"}`

Reply:
Oh, right.  Duh.